tests/EntrypointLookupIntegrityDataProviderInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of the Symfony WebpackEncoreBundle package.
+ * (c) Fabien Potencier <fabien@symfony.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\WebpackEncoreBundle\Tests;
+
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+use Symfony\WebpackEncoreBundle\Asset\IntegrityDataProviderInterface;
+
+interface EntrypointLookupIntegrityDataProviderInterface extends EntrypointLookupInterface, IntegrityDataProviderInterface
+{
+}
```

Comment:
Can you explain why this new interface is needed? Just for convenience? To avoid a problem with using `$this->createMock()` and passing an array? Something else?

Reply:
This new interface exists because createMock() only accepts a string as parameter since phpunit 9.0: https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/TestCase.php#L1698

Reply:
Because this is only used in the `TagRendererTest::class` - I would remove the interface here and just add it at the end of the `TestRendererTest.php` file. Using the signature `TestEntrypointLookupIntegrityDataProviderInterface::class` for the interface would also help mitigate someone from accidentally using this interface outside of the test suite.

Reply:
From my POV the tests/ directory (and other files) should be excluded from the release package using .gitattributes.  
Should be done in an other PR though.

About moving the interface directly in the class file, I'm generally against having multiple class/interface/etc declarations in the same file, but I can do it if you insist :)

Reply:
Let's rename it to `TestEntrypointLookupIntegrityDataProviderInterface`. We can ignore the `tests/` directory in another PR, but I'd still prefer that classes for tests *look* like test classes.

Thanks!

Reply:
Done! :)